core/src/main/java/jenkins/websocket/WebSocketSession.java

```diff
@@ -130,6 +131,15 @@ protected void text(String message) {
         }
     }
 
+    @SuppressWarnings("unchecked")
+    protected final void sendBinary(ByteBuffer partialByte, boolean isLast) {
+        try {
+            remoteEndpoint.getClass().getMethod("sendPartialBytes", ByteBuffer.class, boolean.class).invoke(remoteEndpoint, partialByte, isLast);
```

Comment:
Note that unlike the existing overload, this is blocking. Maybe OK in this context.

Reply:
I didn't check behaviour of `BatchMode`.